ci/terraform/main.tf

```diff
@@ -1,12 +1,118 @@
-resource "google_storage_bucket" "build-artifacts" {
+# Create a storage bucket to hold build artifacts
+resource "google_storage_bucket" "build_artifacts" {
   project                     = var.project_id
   name                        = "${var.project_id}-build-artifacts"
   location                    = "US"
   force_destroy               = true
   uniform_bucket_level_access = true
 }
 
-resource "google_cloudbuild_trigger" "unit-tests-on-push" {
+resource "google_storage_bucket" "test_results" {
+  project                     = var.project_id
+  name                        = "${var.project_id}-test-results"
+  location                    = "US"
+  force_destroy               = true
```

Comment:
I think we want this to be `false` to prevent deleting build history

Reply:
Done.